docs/reference/analysis/tokenfilters/keyword-repeat-tokenfilter.asciidoc

```diff
@@ -1,90 +1,164 @@
 [[analysis-keyword-repeat-tokenfilter]]
-=== Keyword Repeat Token Filter
+=== Keyword repeat token filter
+++++
+<titleabbrev>Keyword repeat</titleabbrev>
+++++
 
-The `keyword_repeat` token filter Emits each incoming token twice once
-as keyword and once as a non-keyword to allow an unstemmed version of a
-term to be indexed side by side with the stemmed version of the term.
-Given the nature of this filter each token that isn't transformed by a
-subsequent stemmer will be indexed twice. Therefore, consider adding a
-`unique` filter with `only_on_same_position` set to `true` to drop
-unnecessary duplicates.
+Outputs each token in a stream twice:
+
+* Once as a <<stemmer-keyword,keyword>>, which is excluded from
+  <<stemming,stemming>>
+* Once as a non-keyword, which can be stemmed normally
+
+This allows you to preserve both a stemmed and unstemmed version of each token.
+
+The `keyword_repeat` filter uses Lucene's
+https://lucene.apache.org/core/{lucene_version_path}/analyzers-common/org/apache/lucene/analysis/miscellaneous/KeywordRepeatFilter.html[KeywordRepeatFilter].
+
+[TIP]
+====
+If you only want to exclude specific tokens from stemming, we recommend using
+the <<analysis-keyword-marker-tokenfilter,`keyword_marker`>> token filter. 
+====
```

Comment:
Same as above...I'd drop "we recommend" in favor of the imperative.

Reply:
Removed with db19ac7bca81c19204bb9ea4220ea7e422319107.